zapisy/apps/offer/assignments/templates/assignments/view.html

```diff
@@ -10,8 +10,8 @@ <h1>Przydział zajęć {{ year }}</h1>
         {% endif %}
     </div>
     {% if teachers %}
-        <div class="jumbotron jumbotron-fluid">
-            <div class="container">
+        <div class="px-3 mb-4 bg-light rounded-3">
+            <div class="container-fluid py-2">
                 <ul>
```

Comment:
To miejsce zostało dostosowane do Bootsrap 5, ale uważam że jest niespójne z resztą widoku strony.
Nie lepiej zrezygnować z jumotron?

Jest:
![image](https://github.com/iiuni/projektzapisy/assets/73119351/e3f16133-3bf8-4db0-a1fd-23462a0ec68a)

Proponuję:
![image](https://github.com/iiuni/projektzapisy/assets/73119351/5faf8657-9e0a-437e-96c9-60889852a553)

```suggestion
        <div class="container">
                <h2>Skróty:</h2>
                <ul>
```

Reply:
Wolałbym, żeby zostało jak jest, choć to kwestia przyzwyczajenia pewnie. Przy czym o ile dobrze, że ten pionowy _padding_ nie jest tak monstrualny jak z oryginalnym `jumbotron`em, to teraz jest on za ciasny, przynajmniej z góry (zresztą widać i na oko, i w deklaracjach klas, że jest mniejszy niż dolny).

Reply:
Ta asymetria jednak wynika z ogólnego stylowania elementu `ul`, który ma margines dolny, a nie ma górnego, i była oczywiście obecna wcześniej, tylko mniej widoczna w starym stylowaniu Bootstrapu. Tak że można np. zrobić
```suggestion
        <div class="px-3 mb-4 bg-light rounded-3">
            <div class="container-fluid py-2">
                <ul class="mt-3">
```
ale tradycyjnie nie wiem, czy to nie jest jakiś zgrzyt z powodu, którego nie jestem świadom.